dkms.conf

```diff
@@ -1,7 +1,5 @@
 PACKAGE_NAME="rtl8812AU"
-PACKAGE_VERSION="#MODULE_VERSION#"
+PACKAGE_VERSION="4.3.22"
 BUILT_MODULE_NAME[0]="8812au"
-MAKE="'make'"
-CLEAN="'make' clean"
```

Comment:
Those two lines are not needed, as DKMS provides defaults. For `MAKE`, I believe it's `make KVER=$(uname -r` (or something similar. For `CLEAN`, the default is `make clean`.